static/app/views/performance/browser/resources/utils/useResourcePagesQuery.ts

```diff
@@ -31,7 +31,9 @@ export const useResourcePagesQuery = (
 
   const queryConditions = [
     ...DEFAULT_RESOURCE_FILTERS,
+    'AND (',
```

Comment:
I wonder if this `AND` is even necessary here, if not we can easily add the brackets to the `getResourceTypeFIlter` without any side effects